src/App.jsx

```diff
@@ -14,33 +16,94 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_BY_NAME = 'name';
+const SORT_BY_LENGTH = 'length';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+export const App = () => {
+  const objModifier = (goods, { sortBy, reversed }) => {
+    const modified = [...goods];
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+    if (sortBy === 'name') {
```

Comment:
```suggestion
    if (sortBy === SORT_BY_NAME) {
```

Use your const variables